src/time_advance.cpp

```diff
@@ -138,6 +139,19 @@ adaptive_advance(method const step_method, PDE<P> &pde,
       auto const y1 =
           adaptive_grid.redistribute_solution(y, old_plan, old_size);
       y.resize(y1.size()) = y1;
```

Comment:
```suggestion
      y = adaptive_grid.redistribute_solution(y, old_plan, old_size);
```
Another copy we can avoid.